code_generator/GdkCodeGenerator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -458,6 +481,8 @@ namespace <#= qualifiedNamespace #>
                 }
                 <#= commandDetails.CommandName #>Failure.Clear();
 <# } #>
+
+                SendAuthorityLossImminentAcknowledgement(connection);
```

Comment:
This shouldn't be in send commands. This is it's own thing really.

Reply:
Can this be in the SpatialOSSend system called separately though? I mean instead of having:

Reply:
Yes I think the second option is more correct

Reply:
Ok changed as shown above.